test/clj/rems/test_browser.clj

```diff
@@ -1145,6 +1145,10 @@
 (defn create-catalogue-item []
   (testing "create catalogue item"
     (btu/with-postmortem
+      (test-helpers/create-category! {:actor "owner"
+                                      :category/title {:en "E2E create-catalogue-item category (EN)"
+                                                       :fi "E2E create-catalogue-item category (FI)"
+                                                       :sv "E2E create-catalogue-item category (SV)"}})
```

Comment:
We should extract this as another step `create-category` and include it, to exercise the UI like the other parts do.